schemas/domainBase.json

```diff
@@ -84,13 +84,137 @@
+                    {
+                        "description" : "MultiPoint domain: mandatory composite axis, optional t axis",
+                        "properties" :
+                        { 
+                            "axes" :
+                            {
+                                "properties" :
+                                {
+                                    "composite" : { "$ref" : "/schemas/tupleValuesAxis" },
+                                    "t" : { "$ref" : "/schemas/stringSingleValueAxis" }
+                                },
+                                "required" : [ "composite" ],
+                                "additionalProperties" : false
+                            }
+                        }
+                    }
+                },
+                {
+                    "if": { "properties" : { "domainType" : { "const" : "MultiPointSeries" } } },
+                    "then":
+                    {
+                        "description" : "MultiPointSeries domain: mandatory composite and t axes",
+                        "properties" :
+                        { 
+                            "axes" :
+                            {
+                                "properties" :
+                                {
+                                    "composite" : { "$ref" : "/schemas/tupleValuesAxis" },
+                                    "t" : { "$ref" : "/schemas/stringValuesAxis" }
+                                },
+                                "required" : [ "composite", "t" ],
+                                "additionalProperties" : false
+                            }
+                        }
+                    }
+                },
+                {
+                    "if": { "properties" : { "domainType" : { "const" : "Section" } } },
+                    "then":
+                    {
+                        "description" : "Section domain: mandatory composite and z axes",
+                        "properties" :
+                        { 
+                            "axes" :
+                            {
+                                "properties" :
+                                {
+                                    "composite" :
+                                    {
+                                        "allOf" :
+                                        [
+                                            { "$ref": "/schemas/tupleValuesAxis" },
+                                            {
+                                                "properties" :
+                                                {
+                                                    "values" :
+                                                    {
+                                                        "items" :
+                                                        {
+                                                            "description" : "In a Section, tuples always have 3 values",
```

Comment:
Any reason why you haven't done the same for the other "composite" cases, e.g. in MultiPointSeries?

Reply:
Good point! No, I just noticed the constraint for Section and didn't think to apply elsewhere! I'll do that and will update the PR.